### PR TITLE
Add missing logs to errors

### PR DESCRIPTION
**Pull request by jdaly13**
# Pull Request 

## Description

This scope of this PR is add the appropriate logging to any errors in meetings plugin so adding 
`LoggerProxy.logger.error(message)`
to any try catch statements and .catch promise methods
Fixes # [SPARK-148851](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-148851)

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**Test Configuration**:
* SDK Version
* Node/Browser Version
* NPM Version

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/meetings/request.js [hunk 1]
```diff
@@ -48,6 +48,8 @@ export default class MeetingRequest extends StatelessWebexPlugin {
         })
         .catch((error) => {
           LoggerProxy.logger.error(`MeetingRequest->determineRedirections# failed to get locus details from url: ${url}, reason: ${error}`);
+
```
Comment (mwegman) [inline packages/node_modules/@webex/plugin-meetings/src/meetings/request.js:51]: I take it this wasn't suppose to be an acceptable error. Was there a reason it was ignored before?

File: packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js [hunk 2]
```diff
@@ -315,14 +315,17 @@ pc.setRemoteSessionDetails = (peerConnection, typeStr, remoteSdp, meetingId) =>
...
+        LoggerProxy.logger.error(`Peer-connection-manager:index#setRemoteDescription --> ${error} missing remotesdp`);
+        return Metrics.postEvent({
         event: eventType.REMOTE_SDP_RECEIVED,
         meetingId: meetingId,
         data: {
           canProceed: false,
           errors: [Metrics.generateErrorPayload(2001, true,
           error.name.MEDIA_ENGINE, 'missing remoteSdp')]
-      }}))
+      }})
```
Comment (mwegman) [inline packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js:327]: Indent this.

File: packages/node_modules/@webex/plugin-meetings/src/common/events/util.js [hunk 3]
```diff
@@ -7,7 +9,7 @@ EventsUtil.getEventLog = (args) => {
     argString = JSON.stringify(args);
   }
   catch (e) {
-    // no op
+    LoggerProxy.logger.error(`Events:util#getEventLog --> ${e}`);
```
[the remark this answers is not in the document]
Comment (jdaly13) [reply packages/node_modules/@webex/plugin-meetings/src/common/events/util.js:12]: or do we not care about logging this exception - I can remove

[the remark this answers is not in the document]
Comment (adamweeks) [reply packages/node_modules/@webex/plugin-meetings/src/common/events/util.js:12]: Info should be fine here in the case of a caught error

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js [hunk 4]
```diff
@@ -1895,7 +1895,11 @@ export default class Meeting extends StatelessWebexPlugin {
             media: mediaResponse,
             local: [localStream, localShare]
           }))))
-      .catch((error) => Promise.reject(error));
+      .catch((error) => {
+        LoggerProxy.logger.error('Meeting:index#join --> Error joinMeeting, ', error);
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:1899]:
```suggestion
        LoggerProxy.logger.error('Meeting:index#joinWithMedia --> ', error);
```

File: packages/node_modules/@webex/plugin-meetings/src/meetings/index.js [hunk 5]
```diff
@@ -512,7 +512,7 @@ export default class Meetings extends WebexPlugin {
       return this.meetingInfo.fetchInfoOptions(destination, type)
         // Catch a failure to fetch info options.
         .catch((error) => {
-          LoggerProxy.logger.info(`meetings->create#INFO, unable to determine info options: ${error.message}`);
+          LoggerProxy.logger.error(`meetings->create#INFO, unable to determine info options: ${error.message}`);
```
[the remark this answers is not in the document]
Comment (adamweeks) [reply packages/node_modules/@webex/plugin-meetings/src/meetings/index.js:515]: An info is good here, as we expect for 1:1 calls to not have meeting info from the server, but there's no way to tell it doesn't until the request goes through.

File: packages/node_modules/@webex/plugin-meetings/src/roap/util.js [hunk 6]
```diff
@@ -83,6 +87,7 @@ RoapUtil.setRemoteDescription = (meeting, session) => {
     };
   })
     .catch((err) => {
+      LoggerProxy.logger.error(`roap:util#setRemoteDescription --> ${err}`);
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/roap/util.js:90]:
```suggestion
      LoggerProxy.logger.error(`Roap:util#setRemoteDescription --> ${err}`);
```

File: packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js [hunk 7]
```diff
@@ -148,6 +148,7 @@ MeetingInfoUtil.getSipUriFromHydraPersonId = (destination, webex) => webex.peopl
   }
   throw new ParameterError('Hydra Id Lookup was an invalid hydra person id.');
 }).catch((err) => {
+  LoggerProxy.logger.error(`MeetingInfo:util#MeetingInfoUtil.getSipUriFromHydraPersonId --> getSipUriFromHydraPersonId ${err} `);
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/meeting-info/util.js:151]:
```suggestion
  LoggerProxy.logger.error(`Meeting-info:util#MeetingInfoUtil.getSipUriFromHydraPersonId --> getSipUriFromHydraPersonId ${err} `);
```

To match @abailey-wwt's changes https://github.com/webex/webex-js-sdk/pull/1710/files#diff-c58ab53994c8db2c15c286b8f77fd9acR54

File: packages/node_modules/@webex/plugin-meetings/src/meeting-info/index.js [hunk 8]
```diff
@@ -80,7 +81,11 @@ export default class MeetingInfo extends StatelessWebexPlugin {
       }
 
       return info;
-    }).catch((error) => Promise.reject(error));
+    }).catch((error) => {
+      LoggerProxy.logger.error(`MeetingInfo:index#requestFetchInfo -->  ${error} fetch meetingInfo`);
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/meeting-info/index.js:85]:
```suggestion
      LoggerProxy.logger.error(`Meeting-info:index#requestFetchInfo -->  ${error}`);
```

**Pull request merged**